internal/peermanager/peermanager.go

```diff
@@ -134,41 +136,41 @@ func (pm *PeerManager) ResponseReceived(p peer.ID, ks []cid.Cid) {
 // to discover seeds).
 // For each peer it filters out want-haves that have previously been sent to
 // the peer.
-func (pm *PeerManager) BroadcastWantHaves(ctx context.Context, wantHaves []cid.Cid) {
+func (pm *PeerManager) BroadcastWantHaves(sid uint64, wantHaves []cid.Cid) {
```

Comment:
The PeerManager is now aware of sessions. Turns out we weren't using any of the contexts in these methods so we can remove them.